apps/src/templates/teacherNavigation/LessonMaterialsContainer.tsx

```diff
@@ -0,0 +1,120 @@
+import _ from 'lodash';
+import React, {useState, useMemo, useCallback} from 'react';
+import {useLoaderData} from 'react-router-dom';
+
+import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
+import {getStore} from '@cdo/apps/redux';
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+import i18n from '@cdo/locale';
+
+import ResourceRow from './ResourceRow';
+
+export type Lesson = {
```

Comment:
Thanks for the tip and context!  I think at this point, `Lesson` will only be needed in this file, but I will keep my eyes open for opportunities to add a `types.ts` file in the future... immediately, I think this could eventually be helpful for a `resource` type, but that might come into clearer focus as we make progress on this feature.